pymeasure/instruments/diamondengineering/damsx000.py

```diff
@@ -287,7 +308,7 @@ def degrees2steps(self, degrees):
         x1 = -R*cos(new_angle_rad)+h*sin(new_angle_rad)
         y1 = R*sin(new_angle_rad)+h*cos(new_angle_rad)
         d1 = self.distance(x1, y1, self.x0, self.y0)
-        return round((d1-ds) * self.steps_per_meter)
+        return round((ds-d1) * self.steps_per_meter)
```

Comment:
```suggestion
        return round((d1-ds) * self.steps_per_meter)
```

To keep Y axis motor on the other side wrt to the horn antenna.

Reply:
Changed in a different way in the code